### PR TITLE
[WIP] Gitcoin Help

### PR DESCRIPTION
**Pull request by usmanmuhd**
Fixes: #3080

### CODE REVIEW

**Comment by usmanmuhd** [comment]
The image is not visible in the top-nav. The link works but can't see the icon. How to resolve this?
image

File: app/retail/templates/shared/top_nav.html [hunk 1]
```diff
@@ -50,4 +50,8 @@
     <img src="{% static "v2/images/labs/gitcoinlabs-logo.svg" %}" height="22" alt="" class="d-md-block d-none" >
     <img src="{% static "v2/images/labs/gitcoin_labs_symbol.svg" %}" height="22" alt="" class="d-md-none">
   </a>
+  <a class="{% if request.path|matches:'^\/((labs.*))$' %}selected{% endif %}" href="{% url 'labs' %}">
+    <img src="{% static "v2/images/labs/help-logo.svg" %}" height="22" alt="" class="d-md-block d-none" >
```
[the remark this answers is not in the document]
Comment (usmanmuhd) [reply app/retail/templates/shared/top_nav.html]: Changed it. Still does not show up.

[the remark this answers is not in the document]
Comment (octavioamu) [reply app/retail/templates/shared/top_nav.html]: check google inspector, see if the source is loading or not. debug your code.

**Comment by thelostone-mc** [comment]
@usmanmuhd could update this PR into a draft PR and change it to an active PR only when it's ready for review

**Pull request closed without merging**